### DISCUSSION TITLE
Changing the media directory where the consumed files are stored

### DISCUSSION THREAD

**Discussion by Schrello** [jonaswinkler/paperless-ng#1435] [Support]
Hey, :)

During the installation I specified `/home/$USER/Documents/Paperless` as the directory where the consumed files should be stored.

Now I want paperless-ng to store the scanned files on my mounted nfs `/mnt/Paperless`.

My understanding of the [documentation](https://paperless-ng.readthedocs.io/en/latest/configuration.html) and [Max's block](https://blog.maxwellfarrior.com/2020-07-13/paperless-in-docker-second-steps/) is that I need to add the line `PAPERLESS_MEDIA_ROOT=/mnt/Paperless` in the `docker-compose.env`.

After the change of docker-compose.env I restart docker using` docker-compose restart`.

But paperless-ng keeps saving the files in the old directory.

Can anyone help me?

I am using paperless-ng 1.5 and Docker to run it.

**Comment by bergfex** [comment]
Try `docker-compose` down and `docker-compose up -d`.
With `restart` only the container is stopped and restarted!
